tfexec/cmd_default.go

```diff
@@ -47,6 +48,19 @@ func (tf *Terraform) runTerraformCmd(ctx context.Context, cmd *exec.Cmd) error {
 		return tf.wrapExitError(ctx, err, "")
 	}
 
+	if interruptCh := ctx.Value(interruptContext); interruptCh != nil {
+		exited := make(chan struct{})
+		defer close(exited)
+		go func() {
+			select {
+			case <-interruptCh.(<-chan struct{}):
+				cmd.Process.Signal(os.Interrupt)
```

Comment:
I think SIGINT is not available in Windows and that's what's making the test to fail 🤔

Reply:
I'm not sure why cmd_linux can't be used for macos as well?

Reply:
I've made cmd_linux -> cmd_unix as the implementation should work in all unix systems
https://github.com/hashicorp/terraform-exec/pull/334/commits/44715fe64ecdc5a92e4bf7916c99951df0df3293